### ISSUE TITLE
poudriere image: make_esp_file is always called with 800 which is too small for a FreeBSD 13 loader.efi

### ISSUE THREAD

**Issue by igalic** [freebsd/poudriere#840]
## Describe the bug

avec trying to build a FreeBSD 13+ image, 
we fail at `make_esp_file`, because the makefs call is creating too small a Filesystem for the new loader to fit

`make_esp_file` is always called with 800 for the size parameter. 
that tells me that the parameter should be either dropped, replaced with a global variable or dynamically calculated.

i replaced all occurrences with a generous 900, and now the image builds.

**Comment by igalic** [comment]
from irc

```irc
01:24 <RhodiumToad> e.g.  sz=$(stat -f %z loader.efi)
01:26 <RhodiumToad> and  imgsz=$(( (sz + 1023)/1024 + 16 ))   should be _just_ big enough to work.
01:26 <RhodiumToad> but if I miscalculated, might need to add more than 16
```

**Comment by evadot** [comment]
Fixed in de9abd5885f5256fd20b
Sorry I forgot the Fixes tag.
